packages/data/pubspec.yaml

```diff
@@ -0,0 +1,20 @@
+name: data
+description: A new Flutter package project.
+version: 0.0.1
+
```

Comment:
Publishable packages can't have 'path' dependencies.
add - publish_to: 'none'
and check other pub specs